packages/static-wado-plugins/lib/index.js

```diff
@@ -0,0 +1,19 @@
+const ConfigPoint = require("config-point");
+
+const { importPlugin } = ConfigPoint;
+
+/**
+ * Define all the local plugins here, so they can be loaded dynamically.
+ */
+const { plugins } = ConfigPoint.register({
+  plugins: {
+    readSeriesIndex: "@ohif/static-wado-plugins/lib/readSeriesIndex.plugin.js",
+    studiesQueryByIndex:
+      "@ohif/static-wado-plugins/lib/studiesQueryByIndex.plugin.js",
+  },
+});
+
+const importer = (name) => import(name);
```

Comment:
sounds to me a promising candidate to be placed on utils project

Reply:
Sigh, no, I tried for over an hour to get that to work, but the import context in each location is different, so if the import is to work, it has to be redefined in each package.